### ISSUE TITLE
✨ Smart Available Current

### ISSUE THREAD

**Issue by SizzinSeal** [LemLib/hardware#10]
#### Overview
the current limit returned by the PROS API does not take into account the number of motors connected to the brain, or motor overheats. Our wrappers should.

**Comment by meisZWFLZ** [comment]
How would that work? Should other devices like leds be incorporated too?

**Comment by SizzinSeal** [comment]
No. There is an algorithm in vexOS that limits motor current beyond 8 motors, (or maybe its 10), and also current gets limited when the motor overheats. This is the available current per motor, not the available current from the battery
